cbioportal_etl/scripts/cnv_3_gistic_style.py

```diff
@@ -42,6 +43,18 @@
     help="Table with cbio project, kf bs ids, cbio IDs, and file names",
 )
 
+def resolve_config_paths(config, tool_dir):
```

Comment:
I agree with @sickler-alex to make it it's own little script/utility that you import, OR see if you can import it from when it's defined in another tool and just pull from there each time. Your choice...the latter may end up more complicated